src/cryptoadvance/specter/templates/includes/tx-data.html

```diff
@@ -184,16 +164,21 @@ <h2>{{ _("Transaction details") }}</h2><br>
                         if ("service_id" in spentOutput) {
                             serviceIdAttr = `data-service-id="${spentOutput.service_id}"`;
                         }
+    
+                        var amountAndUnit = formatBtcAmount(spentOutput.value, spentOutput.assetlabel, this.btcUnit == 'sat');
+                        var price = formatPrice(spentOutput.value, spentOutput.assetlabel, this.symbol, this.price); 
```

Comment:
`assetlabel` isn't defined anywhere, no? I'd just pass null or `let assetLabel` and then pass `assetLabel=null`

Reply:
I thought `assetLabel` is filled for Liquid

Reply:
That's true! I made a comment.